packages/react-keytips/src/docs/Spec.md

```diff
@@ -0,0 +1,190 @@
+| `onExitKeytipsMode`  | `EventHandler<OnExitKeytipsModeData>`  |                 | Callback function triggered when keytip mode is exited.         |
+| `onEnterKeytipsMode` | `EventHandler<OnEnterKeytipsModeData>` |                 | Callback function triggered when keytip mode is entered.        |
+
+## useKeytipRef
+
+| Param          | Type                        | Default                                  | Description                                                                                                                                                       |
+| -------------- | --------------------------- | ---------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `positioning`  | `PositioningProps`          | `{ align: 'center', position: 'below' }` | Positioning props to be passed to Keytip.                                                                                                                         |
+| `onExecute`    | `ExecuteKeytipEventHandler` |                                          | Callback function triggered when keytip is activated.                                                                                                             |
+| `onReturn`     | `ReturnKeytipEventHandler`  |                                          | Callback function triggered when the return sequence is pressed.                                                                                                  |
+| `keySequences` | `string[]`                  |                                          | Array of KeySequences which is the full key sequence to trigger this keytip. Should not include initial 'start' key sequence.                                     |
+| `dynamic`      | `boolean`                   |                                          | Whether or not this keytip will have dynamic content: children keytips that are dynamically created (DOM is generated on keytip activation), Menu, Tabs or Modal. |
+| `appearance`   | `normal \| inverted`        | `inverted`                               | Visual appearance of Keytip.                                                                                                                                      |
+| `visible`      | `boolean`                   | `false`                                  | Control the Keytip's visibility programmatically.                                                                                                                 |
+| `content`      | `string`                    |                                          | The text content of the Keytip.                                                                                                                                   |
+
+## Structure
+
+### JSX Tree
+
+```tsx
+<Portal>
+  <Keytip content="A" positioning={{ target }} />
```

Comment:
If targets are different maybe to mention something like `target1`, `target2` and `target3`